test/parameter_test.py

```diff
@@ -437,6 +451,13 @@ class Foo(luigi.Task):
         p = luigi.parameter.EnumParameter(enum=MyEnum)
         self.assertEqual(hash(Foo(args=MyEnum.A).args), hash(p.parse('A')))
 
+    def test_enum_list(self):
```

Comment:
Can you also add a test case where you pass a default value to the new Parameter?